### DISCUSSION TITLE
Test errors with Autocomplete

### DISCUSSION THREAD

**Discussion by d3spis3d** [lookfirst/mui-rff#1175] [Q&A]
**Describe the bug**
When testing a component that uses the `<Autocomplete />` component from `mui-rff` I encounter an error:
```
Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object.

    Check the render method of `AutocompleteWrapper`.

at input (/Users/code/mui-rff-autocomplete-test/node_modules/mui-rff/src/Autocomplete.tsx:76:3)
        at Field (/Users/code/mui-rff-autocomplete-test/node_modules/react-final-form/dist/react-final-form.cjs.js:718:26)
        at name (/Users/code/mui-rff-autocomplete-test/node_modules/mui-rff/src/Autocomplete.tsx:44:10)
        at form
        at ReactFinalForm (/Users/code/mui-rff-autocomplete-test/node_modules/react-final-form/dist/react-final-form.cjs.js:178:20)
```

My test minimal reproduction test case looks like:

```
import { render } from "@testing-library/react"
import {Autocomplete} from "mui-rff"
import { Form } from "react-final-form"


test("renders autocomplete", () => {
    const options = [
        {value: "1", label: "one"},
        {value: "2", label: "two"},
    ]

    const rendered = render(
        <Form
            onSubmit={() => {
            }}
            render={({handleSubmit}) => (
                <form onSubmit={handleSubmit}>
                    <Autocomplete name="test" label="test" options={options}/>
                </form>
            )}
        />,
    )
})
```

Any ideas or suggestions on what's causing the error in my tests would be much appreciated. I checked the tests in this repo to compare against my own, but with no luck.

**To Reproduce**
I've tried to create a minimal reproduction of the issue in https://github.com/d3spis3d/mui-rff-autocomplete-test

1. Clone the repo
2. Run `yarn && yarn test`

**Expected behavior**
`<Autocomplete />` renders in the test without errors.

**Comment by lookfirst** [comment]
Here is an example you can base your code off of: https://github.com/lookfirst/mui-rff/blob/master/test/Autocomplete.test.tsx

It looks like you might be missing some attributes, like initialValues.

**Reply by d3spis3d** [reply]
Thanks for the reply.

I've updated the reproduction to include initialValues and all other options that match your listed test case, but I'm still receiving the error

```
Error: Uncaught [Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object.

    Check the render method of `AutocompleteWrapper`.
```

Any other ideas or suggestions would be appreciated. Thanks

**Comment by lookfirst** [comment]
Hmm... any errors in your IDE?

**Comment by Kchurch2** [comment]
is there any update on this issue? I am facing the same error currently
